src/test/java/io/stargate/sgv2/jsonapi/TestConstants.java

```diff
@@ -40,15 +41,21 @@ public final class TestConstants {
 
   // CommandContext for working on the schema objects above
 
+  public static final DataApiFeatures DEFAULT_API_FEATURES_FOR_TESTS = DataApiFeatures.empty();
```

Comment:
question - how easy / desirable is it to be able to turn on features in a DataApiFeatures class so we can run unit test without needing the things the happen in the DseTestResource class ?

Reply:
I am thinking that using `DseTestResource` reflects actual usage better, and as long as we do not have tons of testing for alternative setting (that is, most tests can use specific default), works for now.

Otherwise we may end up adding test scaffolding that differs from actual running.
This wrt ITs.